src/Sarif.Driver/Sdk/CommonOptionsBase.cs

```diff
@@ -101,12 +101,12 @@ internal static IEnumerable<FilePersistenceOptions> NormalizeFilePersistenceOpti
         public IEnumerable<string> InsertProperties { get; set; }
 
         [Option(
-            "automationId",
+            "automation-id",
```

Comment:
>automation [](http://example.com/codeflow?start=13&length=10)

these aren't highly utilized, otherwise we might ship a release where they are marked as obsolete.